### DISCUSSION TITLE
PrimeNG p-table filterGlobal Input Not Filtering as Expected

### DISCUSSION THREAD

**Discussion by thetachi559** [primefaces/community#179] [PrimeNG]
### Original Datatable
![image](https://github.com/primefaces/community/assets/28739958/027db3f5-cc8b-4f00-bd3e-94d9e13f4f0e)

### PrimeNG Datatable
![image](https://github.com/primefaces/community/assets/28739958/7ef7113e-2c36-475c-8582-569b1405e725)

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can clearly identify the issue?

You can fork the stackblitz template from this link https://stackblitz.com/edit/lbgg41?file=src%2Fapp%2Fapp.module.ts

**Reply by thetachi559** [reply]
I can't really do a stackblitz because of how old the existing is.  I have included the filter function used to do the filtering the way I'm talking about.

![image](https://github.com/primefaces/community/assets/28739958/83989b5f-3c7d-44e3-97ba-1073016b537b)

**Reply by cetincakiroglu** [reply]
Hi,

Screenshots don't help in this case to understand and identify the issue. 
Could you please fork this [Table StackBlitz Example](https://stackblitz.com/edit/lbgg41?file=src%2Fapp%2Fapp.module.ts) by pressing the fork button located in the upper left corner, then implement your code into the example and share it back with us so we can see the problem and identify it clearly?

**Comment by thetachi559** [comment]
If you go to this link: [https://l-lin.github.io/angular-datatables/#/basic/zero-config](https://l-lin.github.io/angular-datatables/#/basic/zero-config) it will have the search that I'm referencing. If you type "s y" it will filter the way I'm requesting/expecting the primeng table filter does.

The master fork for the project is [https://github.com/l-lin/angular-datatables/tree/master](https://github.com/l-lin/angular-datatables/tree/master)

**Reply by cetincakiroglu** [reply]
Hi,

Could you please reproduce the issue and provide a stackblitz example from the link I've shared above?

**Comment by cetincakiroglu** [comment]
Hi,

We've addressed this issue and probably it's fixed within the 16.8, could you please check it?

The issue is here https://github.com/primefaces/primeng/issues/14016
